heareval/embeddings/runner.py

```diff
@@ -18,16 +19,33 @@
     help="Location of model weights file",
     type=click.Path(exists=True),
 )
+@click.option("--version", default="", help="Version of module and weights")
```

Comment:
I don't get this either. Where is this spec'ed

Reply:
This is specified in the todo at two places. Actually, this is just for convenience. For example, someone has one version of the model and produces embedding for that. And then retrains the model and has another set of weights. This will help them have versions for that. Basically, in the current flow -> Product embeddings which are for a model version, specify the version in the command call, otherwise there will no be no version as it was previously. That is the only thing this does.